Task 2/2.2/Bonuses.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _2._2
+{
+    public class Health : Bonus
```

Comment:
Call it HealthKit or something - so it doesn't look like a class for managing health in character models